mesonbuild/linkers/linkers.py

```diff
@@ -790,7 +790,7 @@ def build_rpath_args(self, env: 'Environment', build_dir: str, from_dir: str,
 
 class GnuDynamicLinker(GnuLikeDynamicLinkerMixin, PosixDynamicLinkerMixin, DynamicLinker):
 
-    """Representation of GNU ld.bfd and ld.gold."""
+    """Representation of GNU ld.bfd, ld.gold, and mold."""
```

Comment:
I'd rather not add mold here, the fact that mold acts so much like bfd and gold it can inherit that class directly is more of an implementation detail. I'm not entirely sure that it's safe to do that though, there may be cases where we use `isinstance(ld, GnuDynamicLinker)` that we don't want to find mold. It might makes more sense to copy the `LLVMDynaicLinker` hierarchy here, though we don't need to override anything it seems.

Reply:
Reverted.